std/algorithm/comparison.d

```diff
@@ -479,6 +479,15 @@ unittest
 Returns $(D val), if it is between $(D lower) and $(D upper).
 Otherwise returns the nearest of the two. Equivalent to $(D max(lower,
 min(upper,val))).
+
+Params:
+    val = The value to clamp.
+    lower = The lower bound of the clamp.
+    upper = The upper bound of the clamp.
+
+Returns:
+    One of the three values as described in the description.
```

Comment:
I don't like this. I think the entire first paragraph should be turned into a `Returns:` block, since that's essentially what it is.